playbooks/common/openshift-cluster/upgrades/pre/verify_cluster.yml

```diff
@@ -41,6 +41,16 @@
     - openshift_release is defined
     - not (openshift_release is version_compare(openshift_upgrade_target ,'='))
 
+  - fail:
+      msg: >
+        No schedulable masters found, please make sure at least one master doesn't have 'openshift_schedulable=false'
```

Comment:
`No schedulable masters found, please remove `openshift_schedulable=False` from all of your masters.`